raw_results.txt

```diff
@@ -50,6 +50,59 @@ Framework: mlx
 
 ==================================================================
 
+============================= M1 Max =============================
+
+---- Training a transformers language model ----
+=> Values from three executions
+
+Framework: pytorch
+	Average: 1106.7549295464829s - Median: 1101.3749282211793s
+
+Framework: mlx
+	Average: 752.2592077255249s - Median: 752.2592812234227s
+
+
+---- Training BERT ----
+=> Values from three executions
+
+Framework: pytorch
+	Average: 793.6759963925679s - Median: 793.610111951828s
+
+Framework: mlx
+	Average: 499.343544960022s - Median: 498.0613958835602s
+
+
+---- Whisper inference ----
+=> Values from ten executions
+
+Framework: pytorch 
+	Average: 21.27653947197935s - Median: 21.204530119895964s
+
+Framework: mlx
+	Average: 6.946336317062378s - Median: 6.937196493148804s
+
+
+---- TinyLLama inference ----
+=> Values from ten executions
+
+Framework: pytorch
+	Average: 50.98743650913239s - Median: 47.73779845237732s
+
+Framework: mlx
+	Average: 20.61291913986206s - Median: 20.613165140151978s
+
+
+---- CPU/GPU switch ----
+=> Values from ten executions
+
+Framework: pytorch
+	Average: 251.71098244190216s - Median: 252.021404504776s
+
+Framework: mlx
+	Average: Framework: mlx
```

Comment:
These is an extra line. Would you mind removing it?

Reply:
Removed, is it good to merge now?

Reply:
Sorry, I meant that line 101 `Average: Framework: mlx` is unnecessary. That was the one to be removed. Besides this, it is good to be merged.

Reply:
Cool, removed it 👍